src/runtime/policy/canonical-policy-annotations.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/** Canonical definitions of annotations used for defining policies. */
+export const canonicalPolicyAnnotations = `
+annotation intendedPurpose(description: Text)
+  targets: [Policy]
+  retention: Source
+  doc: 'Description of a policy.'
+
+annotation egressType(type: Text)
+  targets: [Policy]
+  retention: Source
+  doc: 'Type of egress permitted by a policy. Valid values are: Logging, FederatedAggregation.'
+
+annotation allowedRetention(medium: Text, encryption: Boolean)
```

Comment:
should we let annotations have default values as well? would be nice to be able to set encryption default to false.
we don't have encryption support yet, maybe worth commenting on it in the `doc`.

Reply:
That might be nice.

Done.